ui/src/App.vue

```diff
@@ -104,12 +104,12 @@
               />
             </v-container>
             <create-stream-dialog
-                v-if="accounts().length !== 0"
+                v-if="accounts() && accounts().length !== 0"
                 :is-f-e2-terms="preferences && preferences.user && preferences.user.fe2"
                 :account-id="activeAccount().userInfo.id"
                 :server-url="activeAccount().serverInfo.url"
             />
-            <v-container v-if="accounts().length !== 0" fluid>
+            <v-container v-if="accounts() && accounts().length !== 0" fluid>
```

Comment:
you are calling the accounts() method a lot. Would it be a good idea to call that on load and then save the result? Not a blocker just a suggestion.